packages/visx-responsive/src/hooks/useParentSize.ts

```diff
@@ -0,0 +1,36 @@
+import { RefObject, useRef, useEffect } from 'react';
+import ResizeObserver from 'resize-observer-polyfill';
+import { useSize, WithSizeProps, WithSizeProvidedProps } from './useSize';
+
+export function useParentSize<BaseComponentProps extends WithSizeProps = {}>(
```

Comment:
one thing that we've been considering is using [`react-use-measure`](https://github.com/pmndrs/react-use-measure) to power this instead as it has some more advanced scroll listener functionality built in. we could consider exporting `useMeasure` almost directly for `useParentSize`.

I'm not sure if it could be used for `useScreenSize`, may need to 👀  more closely

Reply:
Great, I'll look it up this weekend.